src/reynir_correct/checker.py

```diff
@@ -456,10 +460,12 @@ def check_with_stats(
     split_paragraphs: bool = False,
     progress_func: ProgressFunc = None,
     annotate_unparsed_sentences: bool = True,
+    **options: Any,
 ) -> CheckResult:
     """Return a dict containing parsed paragraphs as well as statistics"""
     return check_with_custom_parser(
         text,
+        options=options,
```

Comment:
If this is kept unchanged, then options=options should at least come at the end of the parameter list.

Reply:
This has been adressed in Pétur's commit.